helm/g2p-sandbox-security-fynarfin-SIT/Chart.yaml

```diff
@@ -8,4 +8,4 @@ appVersion: 1.0.0
 dependencies:
 - name: ph-ee-g2psandbox-security
   repository: https://fynarfin.io/images/ph-ee-g2psandbox-security
-  version: 0.0.0
\ No newline at end of file
+  version: 1.4.0
```

Comment:
Use the latest version of the chart here., in the master branch we want to keep all version as dev version or latest ( 0.0.0)